src/components/forms/SignUpForm/index.tsx

```diff
@@ -12,8 +15,15 @@ function SignUpForm() {
     getValues,
     formState: { errors }
   } = useForm<User>();
-  // eslint-disable-next-line no-console
-  const onSubmit = () => console.log(getValues());
+  const [errorMsg, setErrorMsg] = useState('');
+  const { isLoading, isError, mutate } = useMutation((data: User) => signUp(data), {
+    // eslint-disable-next-line no-console
+    onSuccess: res => console.log('res', res),
```

Comment:
Add braces to onSuccess handler since there's no intention to return anything, just to execute a console.log
```suggestion
    onSuccess: res => {
      // eslint-disable-next-line no-console
      console.log('res', res);
    },
```

Reply:
great catch!